recipe/meta.yaml

```diff
@@ -9,19 +7,19 @@ package:
 
 source:
   url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-{{ version.replace(".rc", "rc") }}.src.tar.xz
-  sha256: {{ sha256 }}
+  sha256: 11931aadfd80abf2b2643ce9f597a2c8b6b37b0a1574642debcd4dbdaecba889
   patches:
-    #- patches/intel-D47188-svml-VF.patch    # Fixes vectorizer and extends SVML support
-    #- patches/expect-fastmath-entrypoints-in-add-TLI-mappings.ll.patch # adjusts test added in 10.0.0 for intel-D47188-svml-VF.patch effects
-    # Revert an upstream change for Numba
-    # ref: https://github.com/numba/llvmlite/blob/v0.29.0/conda-recipes/0001-Revert-Limit-size-of-non-GlobalValue-name.patch
-    - patches/numba-0001-Revert-Limit-size-of-non-GlobalValue-name.patch
```

Comment:
What happened to this patch?

Reply:
It doesn't apply anymore. There's no `NonGlobalValueMaxNameSize` left in https://github.com/llvm/llvm-project/blob/release/13.x/llvm/lib/IR/Value.cpp

Reply:
It is in function.cpp. See https://github.com/llvm/llvm-project/commit/8d25762720660aba3344752e577ae7017e6125c2

Reply:
OK, thanks for the info. To be honest, I'm not sure I fully understand what's necessary for this (and think it should be done when the other numba-patches are adapted, cf. #123). Is the following the patch as you would envision it?

```diff
diff --git a/llvm/lib/IR/Function.cpp b/llvm/lib/IR/Function.cpp
index 4034b1505bd0..26600bd2d9b9 100644
--- a/llvm/lib/IR/Function.cpp
+++ b/llvm/lib/IR/Function.cpp
@@ -77,10 +77,6 @@ using ProfileCount = Function::ProfileCount;
 // are not in the public header file...
 template class llvm::SymbolTableListTraits<BasicBlock>;

-static cl::opt<unsigned> NonGlobalValueMaxNameSize(
-    "non-global-value-max-name-size", cl::Hidden, cl::init(1024),
-    cl::desc("Maximum size for the name of non-global values."));
-
 //===----------------------------------------------------------------------===//
 // Argument Implementation
 //===----------------------------------------------------------------------===//
@@ -390,7 +386,7 @@ Function::Function(FunctionType *Ty, LinkageTypes Linkage, unsigned AddrSpace,

   // We only need a symbol table for a function if the context keeps value names
   if (!getContext().shouldDiscardValueNames())
-    SymTab = std::make_unique<ValueSymbolTable>(NonGlobalValueMaxNameSize);
+    SymTab = std::make_unique<ValueSymbolTable>();

   // If the function has arguments, mark them as lazily built.
   if (Ty->getNumParams())
```